apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/block-inserter/post-content-block-appender.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { Inserter } from '@wordpress/editor';
```

Comment:
`<Inserter />` has been moved to `@wordpres/block-editor` and it's triggering a deprecation warning.

```suggestion
import { Inserter } from '@wordpress/block-editor';
```

Reply:
https://github.com/Automattic/wp-calypso/pull/34425#discussion_r300818242